### ISSUE TITLE
Quick Start v2 VoiceOver: "Customize Your Site" screen cells should be buttons

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#11492]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. If you already have a site with Quick Start setup, go to that site, double tap on "Customize Your Site", and skip to step 7.
1. Log out of the WPiOS app.
1. Go to WPiOS and double tap the "Sign up for WordPress.com" button.
1. Follow the prompts until you reach "Want a little help getting started?"
1. Double tap the "Yes, Help Me" button.
1. Try interacting with any of the cells on the "Customize Your Site" screen.

Result: the cells on the "Customize Your Site" screen are not buttons. ([19s](https://cloudup.com/crPOM352bDs))

<img src="https://user-images.githubusercontent.com/1119271/56327505-d4098500-6137-11e9-9046-e7097542f873.png" width="270" alt="quick-start-v2-customize-your-site-screen">
<p style="clear:both;"><sup>Tested with WP Internal 12.2.0.20190412 on iPhone 6S iOS 12.2.</sup></p>

<blockquote>It would also be good to set a hint on what will happens after activating the tour. Or maybe some up front explanation as table section header/footer. It might be obvious for most users, but extremely helpful for VoiceOver users.</blockquote>

<p style="clear:both;"><sub>(internal reference: p77Llu-bQ6-p2 h/t @etoledom)</sub></p>
